src/oscar/templates/oscar/customer/emails/commtype_product_alert_body.html

```diff
@@ -0,0 +1,30 @@
+{% extends "customer/emails/base.html" %}
+{% load i18n %}
+
+{% block body %}
+    {% autoescape off %}
+    <p>
+        {% if alert.user and alert.user.get_short_name %}
+            {% blocktrans with name=alert.user.get_short_name %}Dear {{ name }},{% endblocktrans %}
+        {% else %}
+            {% trans "Hello," %}
```

Comment:
Think you need individual `p` tags for each line, otherwise this will all render as one paragraph.